README.md

```diff
@@ -10,110 +10,73 @@ If you are running your own event you've come to the right repo. The documentati
+
+| Archived Tools | Description | Status | Language |
+|----------------|-------------|--------|----------|
 | [EPA Search Utilities](https://github.com/edgi-govdata-archiving/epa-search-utils) | A scraper for the EPA search engine, that systematically feeds in search queries and extracts resultant URLs |  **Experimental**  | Go, Binary |
-| [EPA Geoportal Database Scraper](https://github.com/edgi-govdata-archiving/epa-geoportal-database-scraper) |  | **Working** | Node |
-| [Sitemapper](https://github.com/edgi-govdata-archiving/sitemapper) |  | **Experimental** | Python 3 |
+| [EPA Quantitative Databases](https://github.com/edgi-govdata-archiving/epa-quantitative) | Undocumented scraper for a set of databases accessible but obfuscated through one of the EPA data websites |  **Unknown**  | Python |
+| [Sitemapper](https://github.com/edgi-govdata-archiving/sitemapper) | Tools and services to create xml, csv and json sitemaps of websites  | **Experimental** | Python 3 |
+| [~~EPA ECHO Scraping~~](https://github.com/edgi-govdata-archiving/epa-echo) | Scraper for the EPA Enforcement & Compliance History archives | **Archived** | Ruby |
+| [~~EPA Geoportal Database Scraper~~](https://github.com/edgi-govdata-archiving/epa-geoportal-database-scraper) | Scraper to archives all GIS data ZIP files on EPA's Geoportal | **Archived** | Node |
 | [~~EPA Sitemap~~](https://github.com/edgi-govdata-archiving/epa-sitemap) | A sitemap tool to provide initial models of government domains--intended to facilitate volunteer organization at archivathons |  **Archived**  | Python |
 | [~~EIS Scraping~~](https://github.com/edgi-govdata-archiving/epa-eis) | Full workflow for identifying, scraping, and downloading WARCs of eis's (WARC Archiver above is part) | **Archived** | Ruby, Node, Python 2 |
 | [~~Sprint Toolkit~~](https://github.com/edgi-govdata-archiving/eot-sprint-toolkit) | Our organizing documents for the tech group at our archiv-a-thon (potential overlap with the current repo) |  **Archived**  | `--` |
 
-
 Other tools that we're aware of:
 
 - [Dolley Madison](https://github.com/edgi-govdata-archiving/dolley-madison), a PHP script to download all government Github repos
 - [Grab-Site](https://github.com/edgi-govdata-archiving/grab-site), a crawler with cli that also outputs WARCs
 - [WARCprox](https://github.com/edgi-govdata-archiving/warcprox), a proxy with cli for generating WARCs
 - [Python-sitemap](https://github.com/edgi-govdata-archiving/python-sitemap), a mini-crawler that just makes a sitemap of the website
+
+## Projects
+
+Projects are tracked via issues in their own repos and managed through our [Project Tracking Board](https://github.com/edgi-govdata-archiving/overview/projects/2).
+
+### Monitoring Websites
+
+| Tool Name | Description | Status |
+|-----------|-------------|--------|
+| [**Page Freezer Tools**](https://github.com/edgi-govdata-archiving/pagefreezer-tools) | Initial Repo for a Page Freezer filtering toolkit  | **In Progress** |
```

Comment:
parefreezer tools is deprecated, I'd forgotten that I created it. WIll go now and destroy. remove?